h/templates/admin/search.html.jinja2

```diff
@@ -7,9 +7,6 @@
   <p>
     This operation can be very expensive, so ensure it is not run twice at once.
   </p>
-  <p>
-    The background worker will refuse to process any items after 250,000.
-  </p>
```

Comment:
As we no longer pass in 'indexing' or 'task_id' a tonne of this template can evaporate. We don't need the indexing warnings or anything as it doesn't apply any more